src/components/try-hcp-callout/components/try-hcp-callout/index.tsx

```diff
@@ -0,0 +1,38 @@
+import classNames from 'classnames'
+import CardLink from 'components/card-link'
+import {
+	Description,
+	ProductIconHeading,
+	StandaloneLinkContents,
+} from 'components/try-hcp-callout/components'
+import { TryHcpCalloutProps } from 'components/try-hcp-callout/types'
+import s from './try-hcp-callout.module.css'
+
+/**
+ * Renders an HCP themed callout card,
+ * with a mock UI visual on large viewports.
+ * Intended for use with "Try HCP" callouts.
+ */
+export function TryHcpCallout({
+	productSlug,
+	heading,
+	description,
+	ctaText,
+	ctaUrl,
+}: TryHcpCalloutProps) {
+	return (
+		<CardLink className={s.root} ariaLabel={ctaText} href={ctaUrl}>
```

Comment:
TBH I wasn't sure exactly what to set it to - but having some very long text (like combining some text representation of the heading, description, and ctaText) didn't seem right, it felt like it would be repetitive when moving through the various elements (the `<a />` tag itself, and then the other text elements).

Also just peeked at [the existing HCP callout](https://github.com/hashicorp/dev-portal/blob/zs.try-hcp-components/src/views/homepage/components/merchandising-slots/slots/logo-cta-slot/index.tsx#L19) and other `CardLink` use cases to confirm that take.

I should definitely tested with VoiceOver though, then the choice would have been a bit more informed 😅

Reply:
You're totally all good here!! I think this is the way to go. Concatenating different pieces of text would probably be too verbose.

And in terms of comparing the visual experience and the voice control experience -- my _guess_ is that a voice control user might try to say the text of the CTA to click the card. So setting `ariaLabel` to that exact text prevents us from breaking the guessed voice control experience! (Although we definitely data to confirm the guess)

![image](https://user-images.githubusercontent.com/43934258/202762956-12ddcde4-58db-4f29-b356-612f177d77c7.png)

Nicely done! 🙌🏻